### PR TITLE
Validator API

### PR DESCRIPTION
**Pull request by simzzz**
- Adds initial validators to a mongo collection
- Route for posting validators
- Schema for verification

### CODE REVIEW

**Comment by simzzz** [comment]
`adex-models` dependency needs to be updated once the PR combining validator schemas with the new slots/units schemas is merged.

File: status-loop/getChannels.js [hunk 1]
```diff
@@ -1,9 +1,29 @@
 const getRequest = require('../helpers/getRequest')
 const cfg = require('../cfg')
+const db = require('../db')
+
+async function retrieveChannelsFromDb () {
+	const validatorsCol = db.getMongo().collection('validators')
+	return validatorsCol.find({})
+		.toArray()
+		.then((res) => res)
```
Comment (samparsky) [inline status-loop/getChannels.js]: There is no need for `.then((res) => res)`, `toArray()` returns a promise that would be resolved using `await retrieveChannelsFromDb()`

File: status-loop/getChannels.js [hunk 2]
```diff
@@ -1,9 +1,28 @@
...
+	})
+	return uniqueChannels
+}
 
 async function getChannels () {
-	// const lists = хawait Promise.all(cfg.initialValidators.map(url => getRequest(`${url}/channel/list`)))
-	let { channels } = await getRequest(`${cfg.initialValidators[0]}/channel/list`)
+	const validators = await retrieveChannelsFromDb()
+	const allChannels = await Promise.all(validators.map((v) => getRequest(`${v.url}/channel/list`)))
+	const channels = getUniqueChannels(allChannels)
```
Comment (samparsky) [inline status-loop/getChannels.js]: `await` missing here getUniqueChannels is an async function

File: index.js [hunk 3]
```diff
@@ -56,7 +58,7 @@ db.connect()
 		} else {
 			// Not yet necessary for integration tests
 			startStatusLoop()
-			return Promise.resolve()
+			return Promise.resolve(fillInitialValidators())
```
Comment (samparsky) [inline index.js]: No need for `Promise.resolve` since `fillInitialValidators()` returns a promise

**Pull request merged**